public/assets/env.js

```diff
@@ -0,0 +1,10 @@
+const APP_ENV = {
+    urls: {
+        'social:twitter': 'https://twitter.com/OKP4_Protocol',
+        'social:linkedin': 'https://www.linkedin.com/company/okp4-open-knowledge-platform-for/',
+        'social:discord': 'https://discord.com/invite/okp4',
+        'social:medium': 'https://blog.okp4.network',
+        'form:feedback': 'https://okp4.typeform.com/to/TNyFBH72',
+        'about:okp4': 'https://okp4.network',
+      }
+}
```

Comment:
Just a little remark about record's keys syntax, I will have used nesting keys (e.g: `APP_ENV.urls.social.discord`) to allow destructuration usage instead of "bracket picking" that is less readable and manipulable.

Reply:
Let's say I didn't see the need to introduce an additional layer of indirection (e.g., 'social') in the APP_ENV object. All URLs could be at the same level. 

However, you can still destructure the object if you prefer. To do this, you'll need to specify a new variable name for each URL. Alternatively, we could consider using property syntax to avoid this extra step.